examples/kurth/README.rst

```diff
@@ -48,3 +48,54 @@ We run the following script to analyze correctness:
    .. literalinclude:: analysis_kurth.py
       :language: python3
       :caption: You can copy this file from ``examples/kurth/analysis_kurth.py``.
+
+
+.. _examples-kurth-10nC:
```

Comment:
@cemitch99 I am a bit confused here - we have three new inputs files for tests in this PR but only document one. Do we need to add more info here?